pages/glossary.md

```diff
@@ -0,0 +1,157 @@
+
+## EIP-712
+
+The [EIP-712](https://eips.ethereum.org/EIPS/eip-712) is an Ethereum Improvement Proposal that proposes a standard for hashing and signing of typed structured data as opposed to just bytestrings.
+
+## ERC-4337
+
+The [ERC-4337](https://eips.ethereum.org/EIPS/eip-4337) is an [Ethereum Request for Comments](https://eips.ethereum.org/erc) that introduces a higher-layer pseudo-transaction object called `UserOperation`. Users send `UserOperation` objects into a separate mempool. A special class of actor called bundlers package up a set of these objects into a transaction making a `handleOps` call to a special contract, and that transaction then gets included in a block.
+
+See also:
+- [ERC-4337 Documentation](https://www.erc4337.io) on erc4337.io
+
+## Externally-Owned Account
+
+An externally-owned account (also known as EOA) is one of the two types of Ethereum accounts. It's controlled by a private key, it has no code and users can send messages by creating and signing Ethereum transactions.
+
+See also:
+- [Ethereum Accounts](https://ethereum.org/en/developers/docs/accounts) on ethereum.org
+- [Ethereum Whitepaper](https://ethereum.org/en/whitepaper/#ethereum-accounts) on ethereum.org
+
+## Gasless Transaction
+
+Gasless transactions (also known as meta-transactions) are Ethereum transactions that are executed by a third party called [relayer](#relayer) on behalf of a [smart account](#smart-account) to abstract the use of gas. Users need to sign a message (instead of the transaction itself) with information about the transaction they want to execute and a relayer will be in charge of creating the Ethereum transaction, signing it and executing it, paying for the gas costs. The main benefit is that users can interact with the blockchain without needing to hold the native token in their account.
+
+See also:
+- [Relay Kit documentation](./sdk-relay-kit) on docs.safe.global
+
+## Network
+
+A blockchain network is a collection of interconnected computers that utilize a blockchain protocol for communication.
+Decentralized networks allow users to send transactions, that are processed on a distributed ledger with a consensus mechanism ensuring the batching,
+verification, and acceptance of data into blocks. This structure enables the development of applications without the need for a central authority or server.
+
+See also:
+- [Networks](https://ethereum.org/en/developers/docs/networks) on ethereum.org
+
+## Owner
+
+A Safe owner is one of the accounts that control a given Safe, being the only ones that can manage the configuration of a Safe and approve transactions. They can be either [externally-owned accounts](#externally-owned-account) or [smart accounts](#safe-smart-accounts). The [threshold](#threshold) of a Safe defines how many owners need to approve a Safe transaction to make it executable.
```

Comment:
```suggestion
A Safe owner is one of the accounts that control a given Safe, being the only one to manage the configuration of a Safe and approve transactions. They can be either [externally-owned accounts](#externally-owned-account) or [smart accounts](#safe-smart-accounts). The [threshold](#threshold) of a Safe defines how many owners need to approve a Safe transaction to make it executable.
```

Reply:
This is confusing as an owner cannot manage a Safe by themselves. Coordination is needed among them. I would enphasize that only the group of owners can manage the Safe

Reply:
Let me know if this works better:
```suggestion
A Safe owner is one of the accounts that control a given Safe, being the group ow owners the only ones that can manage the configuration of a Safe and approve transactions. They can be either [externally-owned accounts](#externally-owned-account) or [smart accounts](#safe-smart-accounts). The [threshold](#threshold) of a Safe defines how many owners need to approve a Safe transaction to make it executable.
```

Reply:
How about something like this?

A Safe owner is one of the accounts that control a given Safe. Only owners can manage the configuration of a Safe and approve transactions. They can be either [externally-owned accounts](#externally-owned-account) or [smart accounts](#safe-smart-accounts). The [threshold](#threshold) of a Safe defines how many owners need to approve a Safe transaction to make it executable.